geopandas/geodataframe.py

```diff
@@ -518,6 +587,24 @@ def to_json(self, na="null", show_bbox=False, **kwargs):
         - ``drop``: remove the property from the feature. This applies to each
           feature individually so that features may have different properties.
         - ``keep``: output the missing entries as NaN.
+
+        Examples
```

Comment:
sidenote: I am know you are only adding examples, but seeing this, we should also add a more clearer "see also" mention to `to_file(.., driver="GeoJSON")` to actually write geojson files instead of getting an in-memory json string (which can also be useful of course)

Reply:
"See also" linking is on to-do list :).